konsave

```diff
@@ -10,38 +10,72 @@
 import os, shutil, argparse
 from zipfile import ZipFile, ZIP_DEFLATED, is_zipfile
 
+try:
+    import yaml
+except ModuleNotFoundError:
+    raise ModuleNotFoundError("Please install the module PyYAML using pip: \n"
+                              "pip install PyYAML")
 
 ## GLOBAL VARIABLES ##
 HOME = os.path.expanduser('~')
 CONFIG_DIR = os.path.join(HOME, '.config')
+CONFIG_FILE = os.path.join(CONFIG_DIR, 'konsave', "conf.yaml")
 PROFILES_DIR = os.path.join(CONFIG_DIR, 'konsave/profiles')
-folder_names = ['gtk-2.0', 'gtk-3.0', 'gtk-4.0', 'Kvantum', 'latte']
-file_names = ['dolphinrc', 'konsolerc', 'kcminputrc', 'kdeglobals', 'kglobalshortcutsrc', 'klipperrc', 'krunnerrc', 'kscreenlockerrc', 'ksmserverrc', 'kwinrc', 'kwinrulesrc', 'plasma-org.kde.plasma.desktop-appletsrc', 'plasmarc', 'plasmashellrc', 'gtkrc', 'gtkrc-2.0', 'lattedockrc', 'breezerc', 'oxygenrc', 'lightlyrc', 'ksplashrc']
 export_extension = '.knsv'
 
 
 ## FUNCTIONS ##
 def mkdir(path):
-    '''
+    """
     Creates directory if it doesn't exist
-    '''
+    """
     if not os.path.exists(path):
         os.makedirs(path)
 
 
+def load_config():
+    """
+    Load the yaml file which contains the files and folder to be saved
+
+    The file should be formatted like this:
+
+    ---
+    entries:
+        - folder name
+        - file name
+        - another file name
+        - another folder name
+
+    """
+    if not os.path.exists(CONFIG_FILE):
+        default_entries = ['gtk-2.0', 'gtk-3.0', 'gtk-4.0', 'Kvantum', 'latte', 'dolphinrc', 'konsolerc', 'kcminputrc',
+                           'kdeglobals', 'kglobalshortcutsrc', 'klipperrc', 'krunnerrc', 'kscreenlockerrc',
+                           'ksmserverrc', 'kwinrc', 'kwinrulesrc', 'plasma-org.kde.plasma.desktop-appletsrc',
+                           'plasmarc', 'plasmashellrc', 'gtkrc', 'gtkrc-2.0', 'lattedockrc', 'breezerc', 'oxygenrc',
+                           'lightlyrc', 'ksplashrc']
```

Comment:
I think we could download the config file for them instead of creating a new default_entries list. Here's what we can do:
```
import urllib.request
url = URL_OF_THE_CONF_FILE
destination = PATH_TO_DOWNLOAD_LOCATION
urllib.request.urlretrieve(url, destination)
```

`urllib` is a built in module so there's no problem in using it. This would improve the user experience.